src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -227,7 +227,7 @@ void nextContainer() throws IOException, IllegalArgumentException,
      * @param refIndex reference sequence index
      * @param pos      alignment start to skip to
      */
-    public boolean jumpWithinContainerToPos(final int refIndex, final int pos) {
+    public boolean jumpToAlignment(final int refIndex, final int pos) {
```

Comment:
Should be renamed to reflect the fact that it iterates rather than jumps directly (as mentioned above).